app/javascript/components/modals/Login.js

```diff
@@ -33,13 +33,16 @@ class Login extends Component {
       <Modal visible={!this.props.currentUser}>
         <div className='Login'>
           <div className='Login__header'>
-            Use your UI email address to sign in
+            Sign in using your email address
           </div>
 
           <form onSubmit={this.handleSubmit}>
             <div className='FormInput'>
-              <input type='text' value={this.state.email} onChange={this.handleChange} />
+              <input type='text' placeholder="example@email.com" value={this.state.email} onChange={this.handleChange} />
             </div>
+            <div className='Login__support'>
```

Comment:
Along with the style move, let's move this inside the `FormInput` block, after the input.